Makefile

```diff
@@ -10,20 +10,27 @@ all: verify
 verify: $(PKG_ID).s9pk
 	embassy-sdk verify s9pk $(PKG_ID).s9pk
 
-install: all
+install: $(PKG_ID).s9pk
 	embassy-cli package install $(PKG_ID).s9pk
 
 clean:
+	rm -rf docker-images
 	rm -f image.tar
 	rm -f $(PKG_ID).s9pk
 	rm -f scripts/*.js
 
 scripts/embassy.js: $(TS_FILES)
 	deno bundle scripts/embassy.ts scripts/embassy.js
 
-image.tar: Dockerfile docker_entrypoint.sh assets/utils/*
-	docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/arm64 -o type=docker,dest=image.tar .
+docker-images/x86_64.tar: Dockerfile docker_entrypoint.sh
+	mkdir -p docker-images
+	docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/amd64 --build-arg PLATFORM=amd64 -o type=docker,dest=docker-images/x86_64.tar .
 
-$(PKG_ID).s9pk: manifest.yaml docs/instructions.md icon.png LICENSE scripts/embassy.js image.tar
-	embassy-sdk pack
+docker-images/aarch64.tar: Dockerfile docker_entrypoint.sh
+	mkdir -p docker-images
+	docker buildx build --tag start9/$(PKG_ID)/main:$(PKG_VERSION) --platform=linux/arm64 --build-arg PLATFORM=arm64 -o type=docker,dest=docker-images/aarch64.tar .
 
+$(PKG_ID).s9pk: manifest.yaml instructions.md LICENSE icon.png scripts/embassy.js docker-images/aarch64.tar docker-images/x86_64.tar
+	if ! [ -z "$(ARCH)" ]; then cp docker-images/$(ARCH).tar image.tar; fi
```

Comment:
can be deleted now since was just needed for compatibility with 032 sdk

Reply:
removed. just line 34, correct?